README.md

```diff
@@ -1,3 +1,53 @@
 # curriculum-cr-training-mobile-menu
 
 
+
+# My projects and about section 
+
+> This is a mobile menu project whose purpose is to do some practice of code review.
+
```

Comment:
- [ ] [OPTIONAL] It is better to add a screenshot for your project to inform anyone who opens your repo with what you built till now.